tests/frame/drop_test.py

```diff
@@ -55,6 +42,19 @@ def test_drop_strict(
 
     df = nw.from_native(constructor(data))
 
+    context: Any
+    if strict:
+        if "polars_lazy" in str(constructor):
+            msg = r"^\"z\""
+        else:
+            msg = (
+                r"The following columns were not found: \[.*\]"
+                r"\n\nHint: Did you mean one of these columns: \['a', 'b'\]?"
+            )
```

Comment:
We use `parse_columns_to_drop` and therefore raise our `ColumnNotFoundError.from_missing_and_available_column_names(missing_columns=missing_columns, available_columns=cols)` for every backend